app/serializers/activitypub/note_serializer.rb

```diff
@@ -68,7 +68,7 @@ def cc
   end
 
   def virtual_tags
-    object.mentions.to_a.sort_by(&:id) + object.tags + object.emojis
+    object.mentions.to_a.reject(&:silent?).sort_by(&:id) + object.tags + object.emojis
```

Comment:
`object.active_mentions.to_a`

Reply:
I don't want to touch the eager loading code. Mentions is eager loaded, so it's easier to filter the array than mess with the query causing a n+1

Reply:
active_mentions could also be eager loaded, i think.